packages/react/src/components/Details/Details.stories.tsx

```diff
@@ -1,69 +1,69 @@
 import type { Meta, StoryFn } from '@storybook/react';
 import { useState } from 'react';
 
-import { Button, Link } from '../';
-
-import { Accordion } from '.';
+import { Button, Card, Details, Link } from '../';
 
 export default {
-  title: 'Komponenter/Accordion',
-  component: Accordion,
+  title: 'Komponenter/Details',
+  component: Details,
   parameters: {
     layout: 'padded',
   },
 } as Meta;
 
-export const Preview: StoryFn<typeof Accordion> = (args) => (
-  <Accordion {...args}>
-    <Accordion.Item>
-      <Accordion.Heading>
+export const Preview: StoryFn<typeof Details> = (args) => (
+  <>
+    <Details {...args}>
+      <Details.Summary>
         Hvem kan registrere seg i Frivillighetsregisteret?
-      </Accordion.Heading>
-      <Accordion.Content>
+      </Details.Summary>
+      <Details.Content>
         For å kunne bli registrert i Frivillighetsregisteret, må organisasjonen
         drive frivillig virksomhet. Det er bare foreninger, stiftelser og
         aksjeselskap som kan registreres. Virksomheten kan ikke dele ut midler
         til fysiske personer. Virksomheten må ha et styre.
-      </Accordion.Content>
-    </Accordion.Item>
-    <Accordion.Item>
-      <Accordion.Heading>
+      </Details.Content>
+    </Details>
+    <Details {...args}>
+      <Details.Summary>
         Hvordan går jeg fram for å registrere i Frivillighetsregisteret?
-      </Accordion.Heading>
-      <Accordion.Content>
+      </Details.Summary>
+      <Details.Content>
         Virksomheten må være registrert i Enhetsregisteret før den kan bli
         registrert i Frivillighetsregisteret. Du kan registrere i begge
         registrene samtidig i Samordnet registermelding.
-      </Accordion.Content>
-    </Accordion.Item>
-  </Accordion>
+      </Details.Content>
+    </Details>
+  </>
```

Comment:
Since we are already changing this, can we have the preview stort with just one component?

Reply:
Maybe, or is it nice to just show that several is rendered this way together? :)

Reply:
Since it is the primary story showcasing only the component, I think we should only show one. This is what we do on other component pages as well.
@mimarz migjht have some thoughts